extensions/common/proto_util.cc

```diff
@@ -80,6 +80,8 @@ extractNodeFlatBufferFromStruct(const google::protobuf::Struct& metadata) {
       for (const auto& ip : ip_addresses) {
         ip_addrs.push_back(fbb.CreateString(toStdStringView(ip)));
       }
+    } else if (it.first == "PROXY_VERSION") {
+      proxy_version = fbb.CreateString(it.second.string_value());
```

Comment:
This is duplicate with ISTIO_VERSION label I think. Are you expecting some mis-alignment?